superset/assets/package.json

```diff
@@ -79,6 +81,7 @@
     "react-addons-shallow-compare": "^15.4.2",
     "react-alert": "^2.3.0",
     "react-bootstrap": "^0.31.5",
+    "react-bootstrap-slider": "2.0.1",
```

Comment:
Cool, looks suited to support a nice SliderControl eventually.

Reply:
Yeah, it should be straightforward to add the SliderControl — I actually added it at some point, and then removed.

(The latest version of the component is incompatible with other dependencies, which is why I fixed the version to 2.0.1, BTW.)